### PR TITLE
Test/cypress sightings test v2

### PR DESCRIPTION
**Pull request by RosaTheDev**
X Wrote Tests ____ Implemented ____ Reviewed

## Necessary checkmarks:

    [x] All Tests are Passing in all environments

    [] The code will run locally

## Type of change

    [x] New feature
    [] Bug Fix
    [x] Refactor

## Description of Change:

    description: 

- Refactored the tests in home test, and single cryptid test to be more robust thanks to Grace we worked on it together to make sure the tests were more detailed. 

- Created more tests for the all sightings and made sure that it can filter by cryptoid and the contents are where they are supposed to be

    
## Requested feedback: 
    
    I'd like feedback on... 

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [] No Tests have been changed
    [x] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [x] My code has no unused/commented out code
    [] My code has no binding.pry's
    [] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [x] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: cypress/integration/allsightings-test.js [hunk 1]
```diff
@@ -3,33 +3,54 @@ describe('All sightings page cypress test', () => {
...
+  it('Should click on a sighting of a random cryptoid, and view itss information', () => {
+    cy.get('.sightings-button').click()
+    cy.get('\#\\32').click()
+    cy.get('.sighting-photo')
+    cy.get('.sighting-title').contains('Section A-17 Prototype Labs')
+    cy.get('.sighting-location').contains('Location:')
+    cy.get('#sightingDescription')
+  })
+
+  it('Should filter by searched cryptid and see all the sightings of those cryptoids and move back to all the sightings', () => {
```
Comment (krdurrett) [inline cypress/integration/allsightings-test.js:35]: These tests look great!

File: cypress/integration/singleCryptid-test.js [hunk 2]
```diff
@@ -3,19 +3,21 @@ describe('All sightings page cypress test', () => {
   beforeEach(() => {
     cy.visit('http://localhost:3000')
   })
+  
 
   it('Should start on the home page and click on a cryptid', () => {
-    cy.get('.card-img').eq(0).click()
+    cy.get('.home-message').contains('Ok Cryptid')
+    cy.get('\#\\32').click()
```
Comment (krdurrett) [inline cypress/integration/singleCryptid-test.js:10]: Great trick!

**Review by krdurrett — APPROVED** [review]
These test look great thanks Richard!

**Review by CJSpohn — APPROVED** [review]
I think this a great PR in terms of size and scope. I know it's hard to keep PRs small when you're starting a project from scratch and have so much to accomplish but this is perfect.

**Review by jasmyn2244 — APPROVED** [review]
Thanks Richard! This looks awesome!

**Review by GraceGardner — APPROVED** [review]
Looks great! Thanks for doing all of this hard work.

**Pull request merged**